Sources/NetworkProtection/PacketTunnelProvider.swift

```diff
@@ -700,7 +700,6 @@ open class PacketTunnelProvider: NEPacketTunnelProvider {
             }
 
             Logger.networkProtection.error("🔴 Stopping VPN due to no auth token")
-            await cancelTunnel(with: TunnelError.startingTunnelWithoutAuthToken)
 
             // Check that the error is valid and able to be re-thrown to the OS before shutting the tunnel down
             if let wrappedError = wrapped(error: error) {
```

Comment:
Should we also remove this logic and just throw the error?  I'm feeling like it may be a good idea to clean this up and see if the above fix resolves both issues.

I don't think this has resolve the issue - there are still a good number of reports on Sentry.

Reply:
Removed the outdated code.